extensions/oplogPopulator/modules/Connector.js

```diff
@@ -185,6 +188,7 @@ class Connector {
         }
         try {
             await this._kafkaConnect.deleteConnector(this._name);
+            this.emit(constants.connectorUpdatedEvent, this);
```

Comment:
should be sent before actually deleting the connector

Reply:
Either way will have flaws, if we send it before, and deletion fails, we loose track of the retained bucket (and no easy way to handle it as we work with event (same with callbacks))

Reply:
(I will change it before, but just sharing for visibility)